update readme for the simulation

A-Basirat opened the pull request:
### Changes made:
Add a README file for the simulation.

### Note:
The Look-Up-Table part on the README file needs some description.

Commit:
update readme for the simulation

MattMoradi reviewed (approved):
Looks good. I tend to use one less `#` in each of the headers but that's just me. 
@professorg Do you have time to fill out the lookup table generation section?

bena24 committed:
Update README.md

Updating readme to include explanation of how to run it including how to change parameters, and running the lookup table.

professorg commented:
> Looks good. I tend to use one less `#` in each of the headers but that's just me. @professorg Do you have time to fill out the lookup table generation section?

Sorry, missed this since it was approved. I can do that today.
